### ISSUE TITLE
Selecting a screen in Displays -> Arrangement causes screens to appear to overlap

### ISSUE THREAD

**Issue by kbadk** [Eun/DisableMonitor#20]
When holding down the left mouse button on a screen in Arrangement, the red indicator rectangle appears on the main screen, making it seem like the screens are overlapping.

I think a screenshot will make the issue more clear.

![Screenshot of Arrangement](http://i.imgur.com/uOptQUC.jpg)

**Comment by Eun** [comment]
Not having this issue atm. (Have only 2 Monitors)
Did you disabled all other monitors?

**Comment by kbadk** [comment]
No, this happens simply when disabling one monitor. It seems that all monitors are overlapping now and after playing a little around, this also caused issue #10 (remainings of wallpaper on remaining screen) to occur. Since this is all associated with Arrangement, the fix you made for #10 might also fix this.
